### ISSUE TITLE
UI cleanup

### ISSUE THREAD

**Issue by Chronial** [Wulf2k/DaS-PC-MPChan#94]
As discussed in #84:

- [ ] Remove `Expand DSCM`
- [ ] Remember window size between restarts
- [ ] Add a `Settings` tab as the first tab, but keep the `Active` tab selected on startup.
      - [ ] Remember all settings between restarts
      - [ ] Add explanatory text to all settings
      - [ ] Move maximum node count to settings
      - [ ] Move `Join DSCM-Net` to settings
      - [ ] Move `Node Drawing` to settings (add note about hotkey)
- [ ] Move number of active nodes display to `Active` tab, like for dscm-net
- [ ] Create a `Manual` tab that replaces `Favorites` and `Recent`
      - [ ] Move target input and connect button to manual tab
      - [ ] Remove recent nodes list
      - [ ] Move favorites to manual tab
          - [ ] If available, decorate favorites with data from DSCM-Net and connected nodes
          - [ ] Highlight favorites that we are currently connected to

Now that we want to remember more settings, we should maybe do that [the dotnet way](https://msdn.microsoft.com/en-us/library/a65txexh.aspx)?

**Comment by Chronial** [comment]
@Wulf2k when you find the time, give this a look and see whether this is all fine with you.

**Comment by Wulf2k** [comment]
I agree with all of these, but I'm a little fuzzy on how to do the "Manual" tab without it getting cluttered.

What benefit does the dotnet settings method have over using the registry?  In the rare chance that somebody cares about accessing the settings, the registry is fairly accessible.  How would one go about accessing these settings?

**Comment by Chronial** [comment]
> I agree with all of these, but I'm a little fuzzy on how to do the "Manual" tab without it getting cluttered.

Input and button at the top, favorites underneath.

> What benefit does the dotnet settings method have over using the registry?

I think it will be less code – it will automatically load and save, and it also supports storing lists and objects, not just single values

>  In the rare chance that somebody cares about accessing the settings, the registry is fairly accessible. How would one go about accessing these settings?

They are stored in an xml file in `appdata`
